src/py21cmsense/conversions.py

```diff
@@ -4,20 +4,20 @@
 Provides conversions between observing co-ordinates and cosmological co-ordinates.
 """
 
+from __future__ import annotations
+
 import numpy as np
 from astropy import constants as cnst
 from astropy import units as un
 from astropy.cosmology import FLRW, Planck15
 from astropy.cosmology.units import littleh
-from typing import Union
 
 from . import units as tp
 
 # The frequency of the 21cm line emission.
 f21 = 1.42040575177 * un.GHz
 
 
-@un.quantity_input
```

Comment:
Any particular reason you have decided to remove the quantity_input validator here?

Reply:
Yeah, because it doesn't play well with future annotations (since they turn the annotation into a string under the hood). I would like a solution to that but for now I think this is fine?

Reply:
okay yeah totally fine with me, silly thing but I'm sure it'll get fixed one day.